### PR TITLE
Fix creating local tensorflow model data fails with module import and tensorflow versioning

### PR DESCRIPTION
**Pull request by Fanjia-Yan**
## Describe your changes and why you are making these changes
This PR contains two changes regarding local tensorflow data parameter test:
1. Latest Tensorflow does not support Python 3.7. With older tensorflow, it does not have package module `keras.saving.load_model`. Therefore, we change the loading method to `keras.models.load_model` which presents in both new and older version of tensorflow.
2. Local tensorflow model parameter tests fail with our Periodic Integration test. The reason is that we didn't have tensorflow as a part of param docker image. I added the image with latest version of tensorflow and mark the failed test inactive to k8s which should be reverted next release. 
## Checklist before requesting a review
- [ ] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [ ] I have performed a self-review of my code.
- [ ] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [ ] If this is a new feature, I have added unit tests and integration tests.
- [ ] I have run the integration tests locally and they are passing.
- [ ] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [ ] All features on the UI continue to work correctly.
- [ ] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

File: sdk/aqueduct/utils/serialization.py [hunk 1]
```diff
@@ -114,7 +114,7 @@ def _read_local_bytes_content(path: str) -> bytes:
 def _read_local_tf_keras_model(path: str) -> Any:
     from tensorflow import keras
 
-    return keras.saving.load_model(path)
+    return keras.models.load_model(path)
```
Comment (hsubbaraj-spiral) [inline sdk/aqueduct/utils/serialization.py:117]: If this is backwards compatible (works with multiple TF versions), do we need to pin the TF version in the dockerfile?

Comment (Fanjia-Yan) [reply sdk/aqueduct/utils/serialization.py:117]: Lower bounded it at the latest TF release.

Comment (kenxu95) [reply sdk/aqueduct/utils/serialization.py:117]: I think we'll want to just run the parameter operators locally as a long-term solution for K8s.

File: .github/actions/setup-server/action.yml [hunk 2]
```diff
@@ -62,7 +62,7 @@ runs:
 
     - name: Install packages needed for testing
       shell: bash
-      run: python3 -m pip install nltk pytest-xdist tensorflow
```
Comment (Fanjia-Yan) [inline .github/actions/setup-server/action.yml:65]: Remove tensorflow as a requirement for `setup-server` since we are not testing tensorflow models curently.

**Comment by Fanjia-Yan** [comment]
I disabled the integration test for tf_keras parameter artifact. Since we don't have `tensorflow` as dependencies, previewing the artifact and executing the workflow with keras model will fail.

**Review by hsubbaraj-spiral — APPROVED** [review]
